playbooks/tasks/portal-role-task-mongo-check-docker-service.yml

```diff
@@ -0,0 +1,141 @@
+---
+
+# Check MongoDB docker service runs correctly
+# TODO: This ansible playbook task, should be moved to portal role
+
+# Check mongo container is running correctly
+# Outcome:
+# - mongo_service_ok: boolean flag whether mongo service runs as expected 
+
+- name: Include getting wanted docker compose files
+  include_tasks: tasks/portal-docker-compose-files-get-wanted.yml
+
+- name: Include getting present docker compose files
+  include_tasks: tasks/portal-docker-compose-files-get-present.yml
+
+- name: Start mongo service
+  community.docker.docker_compose:
+    project_src: "{{ webportal_dir }}"
+    # Use intersection of wanted files and present files. E.g. wanted
+    # docker-compose.override.yml is wanted, but might not yet be created.
+    files: "{{ webportal_docker_compose_files_wanted | intersect(webportal_docker_compose_files_present) }}"
+    services: mongo
+    state: present
+  register: docker_mongo_result
+  ignore_errors: True
+
+- name: Set mongo service ok flag
+  set_fact:
+    mongo_service_ok: "{{ not (docker_mongo_result.failed | default(False)) }}"
+
+- name: Log docker compose result on error
+  debug:
+    msg: |
+      Starting mongo servise by docker compose failed:
+
+      {{ docker_mongo_result | to_nice_json }}
+  when: not mongo_service_ok
+
+- name: Wait for mongo container initialization before checking status
+  wait_for:
+    timeout: 10
```

Comment:
Follow Up: Is there an active check we can do instead of a bare sleep?

Reply:
Replaced with checking the logs, timestamps.